examples/evm-to-evm-fungible-transfer/README.md

```diff
@@ -70,9 +70,11 @@ Replace the placeholder values in the `.env` file with your own Ethereum wallet
 
 **Note**
 
-To replace default rpc Cronos and Sepolia urls use env variables:
-- `CRONOS_RPC_URL="CRONOS_RPC_URL_HERE"`
-- `SEPOLIA_RPC_URL="SEPOLIA_RPC_URL_HERE"`
+To replace default transfers from Sepolia to Cronos use env variables:
+- `SOURCE_CHAIN_ID="SOURCE_CHAIN_ID_HERE"`
+- `DESTINATION_RPC_URL="DESTINATION_RPC_URL_HERE"`
```

Comment:
shouldn't be `source rpc url`?

Reply:
nope, because SEPOLIA_RPC_URL was never used. and I added the SOURCE_CHAIN_ID env var